analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -420,8 +420,7 @@ public void Finally_ThrowInFinally()
                 "InTry",
                 "InFinally");
 
-            validator.TagStates("InFinally").Should().HaveCount(1)
-                .And.ContainSingle(x => HasNoException(x));
+            validator.TagStates("InFinally").Should().HaveCount(1).And.ContainSingleNoException();
```

Comment:
We can do this everywhere, also in tests like `TryCatchFinally_ThrowInCatch` where we have
```
            validator.TagStates("InFinally").Should().HaveCount(2)
                .And.ContainSingle(x => HasNoException(x))
                .And.ContainSingle(x => HasSystemException(x));
```
it could be 
```
            validator.TagStates("InFinally").Should().HaveCount(2)
                .And.ContainSingleNoException()
                .And.ContainSingleSystemException();
```

Reply:
Yes, we can do this in a lot of places :) The goal of this PR is just to introduce the methods and as I've already mentioned on slack previously, we will call them in a follow-up PR.